tests/foreman/ui/test_virtwho_configure.py

```diff
@@ -0,0 +1,44 @@
+"""Test class for Virtwho Configure UI
+
+:Requirement: Virt-whoConfigurePlugin
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Virt-whoConfigurePlugin
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from fauxfactory import gen_string
+
+from robottelo.decorators import tier2
+from robottelo.virtwho import get_form_data
+
+
+@tier2
```

Comment:
@sghai are we planning to run this tests in our jenkins? I am asking because if we add this test here, it will always fail for us.

Reply:
A existing hypervisor environment is required to run the virtwho cases, virtwho CI system will provision the environment and run the cases, but any suggestion to store the case in robottelo?

Reply:
@eko999 then as I said https://github.com/SatelliteQE/robottelo/issues/7006#issuecomment-499467701 We need to have separate directory under robottelo/tree/master/tests/foreman/ for virt_who tests

Reply:
okay, looking forward to the final decision from @sghai :+1:

Reply:
@eko999 , I think it is more how/which we run the tests on Jenkins CI from robottelo.

We run tests from specific folders under tests/foreman/ - API/CLI/UI and hence probably @ntkathole is asking u to keep the tests in a separate directory.

Reply:
Sure, I will move the test file to tests/foreman/virtwho/, thank you all.